arangod/GeneralServer/AuthenticationFeature.cpp

```diff
@@ -57,7 +57,8 @@ AuthenticationFeature::AuthenticationFeature(application_features::ApplicationSe
       _authenticationSystemOnly(true),
       _localAuthentication(true),
       _active(true),
-      _authenticationTimeout(0.0) {
+      _authenticationTimeout(0.0),
+      _sessionTimeout(60.0 * 60.9 * 24.0 * 30.0 /*30 days*/) {
```

Comment:
can't we use `std::chrono::duration_cast<std::chrono::day>` here?

Reply:
I agree with the sentiment, but `std::chrono::days` is only available since C++20. You could write `std::chrono::hours(30 * 24).count()` or `30 * std::chrono::hours(24).count()`. Otherwise, there's a typo to fix:
```suggestion
      _sessionTimeout(60.0 * 60.0 * 24.0 * 30.0 /*30 days*/) {
```

Reply:
Ah, note that I didn't add the cast into seconds, so with chrono it'd actually have to be
```suggestion
      _sessionTimeout(30 * std::chrono::hours(24) / std::chrono::seconds(1)) {
```
. I'm personally fine with either.